geomstats/geometry/special_orthogonal.py

```diff
@@ -51,18 +51,19 @@ def __init__(self, n, point_type=None, epsilon=0.):
             raise ValueError('Parameter `n` is required to be an integer > 1.')
 
         self.n = n
```

Comment:
We could decide on guidelines to organize the code our constructors, like:
- checks of parameters with geomstats.error.check_*,
- auxiliary computations if needed,
- calls of constructor(s) of parent's class(es), with super(),
- assignment of remaining attributes, with self.XXX = XXX
?